### ISSUE TITLE
Using a library like https://github.com/mfiro/hnconnector

### ISSUE THREAD

**Issue by nicobrenner** [hughdbrown/who-is-hiring#1]
I was checking out the code, and have the impression that ids for the Who's hiring posts are being added manually

It seems like using the hnconnector library, getting the latest could be automated and then you could have a cron running the 1 of the month to find the new post

**Comment by hughdbrown** [comment]
Yes, I have been adding the HN ids manually. No, I did not see this Issue earlier.
